frontend/src/components/FacilityForm/FacilityForm.js

```diff
@@ -61,16 +116,18 @@ const FacilityForm = () => {
                   key={index}
                   icon={facility.icon}
                   name={facility.name}
+                  onLongPress={() => handleLongPressFacility(facility.name)}
                   onPress={() => handleSelectFacility(facility.name)}
-                  isSelected={facilities.includes(facility.name)}
+                  isSelected={wantedFacilities.includes(facility.name)}
+                  isDoubleSelected={requiredFacilities.includes(facility.name)}
                 />
               </View>
             ))}
           </View>
         </View>
         <View>
           <Pressable>
-            <Button onPress={() => handleSubmit()} title={"The old button!"} />
+            <Button onPress={() => handleSubmit()} title={"Continue"} />
```

Comment:
I deliberately changed this to make it clear it was no longer 'the' button. Please undo this change or change to anything that indicates it is the old button that makes sense to you. There is a ticket for moving this functionality to the new buttons in the menu bar.